pom.xml

```diff
@@ -28,11 +28,6 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-rest</artifactId>
         </dependency>
-        <dependency>
```

Comment:
Postgres dependency not needed?

Reply:
We had duplicated postgres dependencies. It was giving an error

Reply:
oh yes I see line 51 now